app/src/main/res/xml/remote_config_defaults.xml

```diff
@@ -144,4 +714,75 @@
+    </entry>
+    <entry>
+        <key>exposureBodyMid</key>
+        <value>Pokud se u vás v průběhu 14 dnů od kontaktu s nakaženým objeví příznaky respiračního onemocnění, kontaktujte telefonicky svého praktického lékaře a řiďte se jeho pokyny. V případě akutních potíží kontaktujte nejbližší pohotovost.
+        </value>
+    </entry>
+    <entry>
+        <key>exposureBodyBottom</key>
+        <value>Jestliže se u vás neprojevují příznaky onemocnění, nemusíte zůstávat doma. Dodržujte zásady bezpečného kontaktu.
+        </value>
+    </entry>
+    <entry>
+        <key>noExposureHeader</key>
+        <value>V posledních 14 dnech nebyla ve vaší blízkosti žádná osoba s potvrzenou nákazou virem
+            COVID-19
+        </value>
+    </entry>
+    <entry>
+        <key>noExposureBody</key>
+        <value>Na rizikové setkání vás upozorníme pomocí oznámení.</value>
+    </entry>
+    <entry>
+        <key>mainSymptoms</key>
+        <value>Hlavní příznaky</value>
+    </entry>
+    <entry>
+        <key>spreadPrevention</key>
+        <value>Zásady bezpečného kontaktu</value>
+    </entry>
+    <entry>
+        <key>earlierExposures</key>
+        <value>Předchozí riziková setkání</value>
+    </entry>
+    <entry>
+        <key>mainSymptomsBody</key>
+        <value>Příznaky se mohou objevit 2-14 dní po rizikovém setkání.</value>
+    </entry>
+    <entry>
+        <key>preventionBody</key>
+        <value>Koronavirus se přenáší hlavně kapénkovou infekcí, která je přenášena vzduchem. Zasahuje nejčastěji sliznice horních a dolních dýchacích cest a spojivky. Přenášet se může také kontaminovanými předměty.
+        </value>
+    </entry>
+    <entry>
+        <key>exposureUITitle</key>
+        <value>Rizikové setkání</value>
+    </entry>
+    <entry>
+        <key>symptomsUITitle</key>
+        <value>Hlavní příznaky</value>
+    </entry>
+    <entry>
+        <key>spreadPreventionUITitle</key>
+        <value>Jak zabránit přenosu</value>
+    </entry>
+    <entry>
+        <key>recentExposuresUITitle</key>
+        <value>Předchozí riziková setkání</value>
+    </entry>
+    <entry>
+        <key>preventionContent</key>
+        <value>[{"iconUrl":"","label":"Často a důkladně si myjte ruce mýdlem či dezinfekčním gelem."},{"iconUrl":"","label":"Pravidelně dezinfikujte také vlastní předměty (např. mobilní telefon)."},{"iconUrl":"","label":"Kašlete a kýchejte do kapesníku či rukávu."},{"iconUrl":"","label":"Používejte jednorázové kapesníky a poté je vyhoďte."},{"iconUrl":"","label":"Vyhýbejte se velkému uskupení osob a udržujte bezpečný odstup (cca 2 metry)."},{"iconUrl":"","label":"Eliminujte kontakt s nemocnými lidmi a necestujte do zasažených lokalit."}]</value>
+    </entry>
+
+    <entry>
+        <key>symptomsContent</key>
+        <value>[{"iconUrl":"","label":"Zvýšená teplota"},{"iconUrl":"","label":"Kašel"},{"iconUrl":"","label":"Dušnost"},{"iconUrl":"","label":"Bolest v krku"},{"iconUrl":"","label":"Bolest hlavy"}]</value>
```

Comment:
Pošli PR na web repo, až se to zamerguje a deployne, tak nahraď URLs tady

Reply:
iconUrls jsou nyní dostupné, kód jsem aktualizoval a ikony jsou v aplikaci nyní viditelné